generators/create-react-app/index.ts

```diff
@@ -38,11 +39,15 @@ class CreateReactAppGenerator extends PackageGenerator {
                 // Update chart
                 this.configureChart('deployment/kubernetes/chart');
 
-                this.replaceDestination(
-                    'modules/deployment/chart/values.yaml',
-                    /$/s,
-                    `\n${varName(packageName)}:\n  image:\n    tag: latest\n    digest: ~\n  sentry:\n    dsn: ~\n`,
-                );
+                this.appendDestination('modules/deployment/chart/values.yaml', indent`
+
```

Comment:
The tag fails to detect indentation because of this empty line, not sure if this should be fixed in the tag or we have to change how we use it.

Reply:
This should be fixed by https://github.com/jvasseur/indent-tag/pull/2